pkg/features/kube_features.go

```diff
@@ -852,6 +852,20 @@ const (
 	// restore the old behavior. Please file issues if you hit issues and have to use this Feature Gate.
 	// The Feature Gate will be locked to true and then removed in +2 releases (1.35) if there are no bug reported
 	DisableCPUQuotaWithExclusiveCPUs featuregate.Feature = "DisableCPUQuotaWithExclusiveCPUs"
+
+	// owner: @munnerz
+	// kep: https://kep.k8s.io/4742
+	// alpha: v1.33
+	//
+	// Enables the PodTopologyLabelsAdmission admission plugin that mutates `pod/binding`
+	// requests by copying the `topology.k8s.io/{zone,region}` and `kubernetes.io/hostname`
```

Comment:
Been thinking about `kubernetes.io/hostname` more and I don't think we should copy it. Mainly for 2 reasons:
1) A user reading `kubernetes.io/hostname` label on a Pod would assume the this label is for the hostname of the Pod, not the Node. 
2) You can get node name today using `spec.nodeName` and this almost always matches `kubernetes.io/hostname`.

Reply:
I've updated the PR to omit copying `kubernetes.io/hostname` for now as I do agree - I'd rather us standardise `hostname` as a form of topology label instead, or otherwise require users to use a custom solution for this.. I don't think there is clear consensus at all on whether a `kubernetes.io/hostname` label is actually routable, especially when you consider things like the konnectivity outbound proxy may typically sit between the apiserver and the dataplane.